.circleci/config.yml

```diff
@@ -97,12 +103,18 @@ commands:
                 working_directory: test/acceptance/tests
                 no_output_timeout: 2h
                 command: |
+                  # Enterprise tests can't run on fork PRs because they require
+                  # a secret.
+                  if [ -z "$CIRCLE_PR_NUMBER"]; then
+                    ENABLE_ENTERPRISE=true
+                  fi
+
                   pkgs=$(go list ./... | circleci tests split --split-by=timings --timings-type=classname)
                   echo "Running $pkgs"
                   gotestsum --junitfile "$TEST_RESULTS/gotestsum-report.xml" -- $pkgs -p 1 -timeout 2h -failfast \
                       << parameters.additional-flags >> \
                       -enable-multi-cluster \
-                      -enable-enterprise \
+                      ${ENABLE_ENTERPRISE:+-enable-enterprise} \
```

Comment:
from https://stackoverflow.com/questions/42985611/how-to-conditionally-add-flags-to-shell-scripts